pkg/controller/installation/prometheusRules.go

```diff
@@ -62,6 +63,23 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 					},
 				},
 			},
+			{
+				AlertName: "rhmi-upgrade-alerts",
+				Namespace: monitoring.OpenshiftMonitoringNamespace,
+				GroupName: "rhmi-upgrade.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHMIUpgradeTooLong",
```

Comment:
Could the name of the alert be more descriptive? Something like `RHMIUpgradeExpectedDurationExceeded`?